tests/test-files/define-type-alias.txt

```diff
@@ -0,0 +1,142 @@
+================================================================================
+1 Define type alias
```

Comment:
Should add tests around package boundaries. Permutations that come to mind:

* use an alias for a standard type from another package that has been imported with use
* Reference an alias for a standard type from another package without `:use`

- Both of the above to a type that is from the same package but was not exported.
- Both of the above, but the alias in package 2 is to a type  from a third package which is not `:use`d in the main package